### DISCUSSION TITLE
Help with Docker Container just for Farming on NAS

### DISCUSSION THREAD

**Discussion by MEBruck** [Chia-Network/chia-blockchain#7075] [General]
I have a full node running on Windows and just wanted to setup my Synology NAS for farming. I was able to get the Docker Container setup of Chia up and running (it recognizes my existing plots, which happen to be stored on the same NAS & my wallet). However, I'm unsure what to do to link this Docker install of Chia with my original main node, which I JUST want to Plot now and view my wallet. I want that main node to stop farming and have that fully taken over by the the Chia running on my NAS. I can't seem to find the steps to finish that part. Any help you can provide would be GREATLY appreciated. I hate to say it, I'm also a Windows-guy NOT a Unix person, so any extra detail on your instructions/steps to help me out there is also appreciated.

**Comment by loppefaaret** [comment]
simply remove the plot from the client
